hardware/views.py

```diff
@@ -0,0 +1,65 @@
+from django.urls import reverse
+from django.views.generic import TemplateView
+from django.shortcuts import render
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.http import JsonResponse
+from user.mixins import IsVolunteerMixin
+from app.mixins import TabsViewMixin
+from user.models import User
+from hardware.models import Item, ItemType, Lending
+
+def hardware_tabs(user):
+    first_tab = ('Hardware List', reverse('hw_list'), False)
+    if user.is_volunteer:
```

Comment:
Do we want to give access to all volunteers? Should we add a new category instead?

Reply:
Yeah I should probably ask to logistics to see how they want to manage this. In previous editions I think that at some point there werer volunteers handling the harware lab. But yeah, organizers or new category will be best. Also I should probably add who handled the lending or the return of an item.